docs/administration-guide/job-tracker.md

```diff
@@ -0,0 +1,138 @@
+# Job Tracker
+
+This section covers how to configure your Gaffer graph to allow jobs to be executed.
+
+'Job' refers to an operation chain with a long-running sequence of operations that is executed
+asynchronously. 
+Jobs are, therefore, useful for operation chains which take a long time to return results or if 
+you want to cache the results of your operation chain. 
+
+Jobs are only available when the job tracker has been enabled. 
+The job tracker can then be used to store the status of current and historic jobs 
+while a 'cache' is used to store any job results. 
+
+## Configuration
+
+By default, the job tracker is disabled. 
+It can be enabled by setting its associated [store property](../administration-guide/gaffer-stores/store-guide.md/#cache-service) to true.
```

Comment:
This is linking to the cache section, I think it should link to the properties list which is where the option to enable job tracker is.
```suggestion
It can be enabled by setting its associated [store property](../administration-guide/gaffer-stores/store-guide.md#all-general-store-properties) to true.
```